plugins/chat/assets/javascripts/discourse/components/chat-composer.js

```diff
@@ -106,13 +100,15 @@ export default class ChatComposer extends Component {
 
   @action
   didUpdateMessage() {
-    cancel(this._persistHandler);
+    this.cancelPersistDraft();
```

Comment:
this and next block, we try to cancel current call and persist as soon as possible